recipe/build.sh

```diff
@@ -2,6 +2,13 @@
 
 export LDFLAGS="-L${PREFIX}/lib"
 export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
+    brew uninstall gmp
```

Comment:
Could we check that we are on Travis CI (e.g. [`TRAVIS`]( https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables ))?